content/guides/core-concepts/cypress-app.md

```diff
@@ -6,29 +6,108 @@ title: Cypress App
 
 ## <Icon name="graduation-cap"></Icon> What you'll learn
 
-- The names and purposes of the visual parts of the Cypress App
+- The names and purposes of the visual parts of the Cypress app
 - How to use the Selector Playground for targeting your page elements
-- How to debug tests using the built-in features of the Cypress App
+- How to debug tests using the built-in features of the Cypress app
 
 </Alert>
 
 ## Overview
 
-Cypress runs tests in a unique interactive application that allows you to see
-commands as they execute while also viewing the Application or Component Under
-Test.
+The Cypress desktop app has powerful features for creating, configuring,
+browsing, and most importantly running your tests. We'll touch on a few of the
+different functions before moving on to the Test Runner, the heart of the
+Cypress testing experience.
 
-<DocsImage src="/img/guides/cypress-app/gui-diagram.png" alt="Cypress App"></DocsImage>
+<Alert type="info">
+
+<strong class="alert-header">Dashboard Integration</strong>
+
+A number of the features described below will only be shown if you are logged
+into the [Cypress Dashboard](/guides/dashboard/introduction). If you're not sure
+whether you're logged in, you can always check using the profile control in the
+top right of the window.
```

Comment:
```suggestion
When you configure Cypress to record tests to the [Dashboard](/guides/dashboard/introduction), you will get data from your latest recorded runs in the Cypress app to increase visibility into your test history. If you're not sure whether you're logged in, you can always check using the profile control in the
top right of the window.
```

Reply:
@JessamynSweet @debrisapron let me know what you think of this wording - still playing with it

Reply:
I think this is pretty good, gonna add this for now